main.c

```diff
@@ -297,6 +298,8 @@ static const struct fuse_opt ovl_opts[] = {
    offsetof (struct ovl_data, max_idle_threads), 0},
   {"mt=%d",
    offsetof (struct ovl_data, mt), 0},
+  {"omitxattr",
+   offsetof (struct ovl_data, omitxattr), 1},
```

Comment:
The default value in that case is zero since the options are `memset` to zero before parsing:
https://github.com/stanford-rc/fuse-migratefs/blob/24949e94d3160107408a0fb343a6b11d4819c66a/main.c#L3916-L3920

The `1` only means that as soon as the `omitxattr` parameter is passed without value, the value `1` is used.
Meaning `-o omitxattr` will set the value 1, while `-o whatever` will not touch the value at all.
If I got the [fuse option parsing](https://github.com/libfuse/libfuse/wiki/Option-Parsing) and my testing right of course.
I can double check tomorrow though since running with an xattr-supporting *lowerfs* and `-f` should print `OMITXATTR=0`.

Reply:
Thanks for this clear explanation, this totally makes sense to me now and I guess you got it right!